app/views/profile_preview_cards/show.json.jbuilder

```diff
@@ -1,6 +1,6 @@
 json.extract!(
   @user.profile,
-  :summary, :employment_title, :employer_name, :employer_url, :location, :education
+  :summary, :location, :education, :work
```

Comment:
@aitchiss `summary`and `location` are ok because they are columns on `profiles` and will therefore always be available on all Forems. The same is not true for `education` and `work`: while we do create them by default (at least for now) admins will be able to remove them. I guess we should do something similar to this, e.g. add all header fields (a maximum of three):

```ruby
      <% if (header_fields = actor.profile.decorate.ui_attributes_for(area: :header)).present? %>
        <% header_fields.sort.each do |title, value| %>
          <% next if Profile.special_attributes.include?(title) %>
          <li>
            <div class="key">
              <%= title %>
            </div>
            <div class="value">
              <%= value %>
            </div>
          </li>
        <% end %>
      <% end %>
```

However, this PR is getting big enough as is, so I'll do that later.